sematic/ui/packages/common/src/theme/new/componnets.ts

```diff
@@ -0,0 +1,32 @@
+import { Components } from '@mui/material/styles';
+
+const components: Components = {
+    MuiButton: {
+        variants: [
+            {
+                props: { variant: 'logo' },
+                style: {
+                    textTransform: 'none',
+                    width: 50,
+                    height: 50,
+                    minWidth: 'min-content'
+                },
+            },
+            {
+                props: { variant: 'menu' },
+                style: ({theme}) => {
+                    return {
+                        color: theme.palette.black.main,
```

Comment:
I thing you should also have `textTransform: 'none"` here as well. The buttons are not capitalized in the design.

Reply:
Indeed. Fixed. Thanks!